cmd/release/cmd/update.go

```diff
@@ -42,8 +44,41 @@ var updateK3sReferencesCmd = &cobra.Command{
 	},
 }
 
+var updateChartsCmd = &cobra.Command{
+	Use:   "charts [branch] [chart] [version]",
+	Short: "Update charts files locally, stage and commit the changes.",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		var branch, chart, version string
+
+		if len(args) < 3 {
+			return errors.New("expected at least three arguments: [branch] [chart] [version]")
+		}
+
+		branch = args[0]
+		chart = args[1]
+		version = args[2]
```

Comment:
You should also check if these args are valid or not

Reply:
Maybe use the semver package that some other commands are already using

Reply:
@tashima42 I have implemented a new strategy for checking the arguments and auto-completion. 
Please review it again.